src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Table.java

```diff
@@ -331,15 +332,29 @@ public void addRow(T item, List<Content> contents) {
             }
         }
         int colIndex = 0;
+        String regex = "<(?:a|area|button|input|object|select|textarea)\\b";
+        Pattern pattern = Pattern.compile(regex);
         for (Content c : contents) {
             HtmlStyle cellStyle = columnStyles.get(colIndex);
+            // Add tab order to only plain text to avoid widget_tabbable_single(potential violation)
```

Comment:
`widget_tabbable_single` sounds like a programmatic name. Can we provide a better explanation of what this rule means, maybe also adding a link? [This piece of code][tabbable-single] seems to provide an explanation, but I'm not sure what it means in the current context.

[tabbable-single]: https://github.com/IBMa/equal-access/blob/7954dcdc5f1b5536057f783abcae1f7521265200/karma-accessibility-checker/test/client/baseline/JSONObjectStructureVerification.html.json#L2810-L2811

Reply:
@hns We're trying to add tab order only to plain elements here so that we can avoid multiple tabbable elements in the tablist, as other elements like link,object,buttons have default tab order. I modified comment as you suggested. Could you please review once. Thank you.